test/libsolidity/interface/FileReader.cpp

```diff
@@ -226,10 +226,10 @@ BOOST_AUTO_TEST_CASE(normalizeCLIPathForVFS_stripping_root_name)
 		BOOST_CHECK_EQUAL(normalizedPath.root_directory(), "/");
 
 #if defined(_WIN32)
-		string root = workDir.root_path().string();
+		std::string root = workDir.root_path().string();
 		soltestAssert(root.length() == 3 && root[1] == ':' && root[2] == '\\', "");
 
-		for (auto convert: {boost::to_lower_copy<string>, boost::to_upper_copy<string>})
+		for (auto convert: {boost::to_lower_copy<std::string>, boost::to_upper_copy<std::string>})
 		{
 			boost::filesystem::path workDirWin = convert(root, locale()) / workDir.relative_path();
```

Comment:
```suggestion
			boost::filesystem::path workDirWin = convert(root, std::locale()) / workDir.relative_path();
```

Windows build is failing due to this.